app/poc-common/form-display/directives/formField.js

```diff
@@ -35,11 +35,20 @@ angular.module('poc.common.formdisplay')
                     if(last.value.datatype && last.value.datatype.display === "Coded") {
                         $scope.fieldModel.value = realValueOfField($scope.fieldModel.fieldConcept.concept.answers, last.value);
                         return;
+                       
                     }
                     $scope.fieldModel.value = last.value; 
                }
             });
         };
+
+        formLogic.seeData = function (data) {
```

Comment:
I don't think I understand what this function does. It just checks a value, but doesn't return anything. What is it's purpose?